src/utils.js

```diff
@@ -4,7 +4,7 @@ import path from 'path';
 import fsPromises from 'fs/promises';
 import fs, { existsSync } from 'node:fs';
 import clone from 'clone';
-import glyphCompose from '@mapbox/glyph-pbf-composite';
+import * as glyphCompose from '@jsse/pbfont';
```

Comment:
Can we just import the named combine function 'combine'

Reply:
Yes, Have made the changes accordingly. :)